docs/site/reference/error-codes.md

```diff
@@ -16,6 +16,7 @@ sets the error `code` property to a machine-readable string.
 | VALIDATION_FAILED          | The data provided by the client is not a valid entity.                                                                                                                                                    |
 | INVALID_PARAMETER_VALUE    | The value provided for a parameter of a REST endpoint is not valid. For example, a string value was provided for a numeric parameter.                                                                     |
 | MISSING_REQUIRED_PARAMETER | No value was provided for a required parameter.                                                                                                                                                           |
+| CANNOT_INFER_PROPERTY_TYPE | See below: [CANNOT_INFER_PROPERTY_TYPE](#cannot_infer_property_type)                                                                                                                                      |
```

Comment:
Maybe we should get rid of this table and create a level2 (`##`) section for each error code? There will be auto-generated table of contents at the top of the page to allow quick lookup of a particular error.

I prefer to make such changes in a different PR, preferably after this one is landed.

Reply:
Opened https://github.com/strongloop/loopback-next/pull/6363